bpf/lib/icmp6.h

```diff
@@ -316,6 +316,7 @@ static __always_inline int __icmp6_send_time_exceeded(struct __ctx_buff *ctx,
 			return DROP_WRITE_ERROR;
 
 		break;
+	/* FIXME: Add IPPROTO_SCTP support */
```

Comment:
We should create an issue for this instead of leaving a note in the code. I think it can be an issue for all STCP leftovers if there are several.

Reply:
I think we can.

Reply:
^^ If you are going to continue pursuing this @DolceTriade, then maybe it would be worthwhile for you to create an issue for it so that you can edit / update the status in the issue description. We can then point this one to the new one you create.

Reply:
Created https://github.com/cilium/cilium/issues/20490